frontend/templates/troubleshooting/index.tsx

```diff
@@ -62,11 +68,53 @@ function ConclusionSection({ conclusion }: { conclusion: Problem }) {
    return (
       <Box>
          <Heading>{conclusion.heading}</Heading>
-         <Box dangerouslySetInnerHTML={{ __html: conclusion.body }} />
+         <Prerendered html={conclusion.body} />
       </Box>
    );
 }
 
+const Prerendered = chakra(function Prerendered({
+   html,
+   className,
+}: {
+   html: string;
+   className?: string;
+}) {
+   const renderStyles: SystemStyleObject = {
+      '.headerContainer': {
+         display: 'flex',
+         alignItems: 'baseline',
+         marginBottom: 2,
+      },
```

Comment:
Do you know if chakra automagically does scoping?

i.e. `& .headerContainer`?

Reply:
I believe it does:
```css
.css-eec6dr .headerContainer {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
  margin-bottom: var(--chakra-space-2);
}
```
(from the rendered page)